### PR TITLE
index ARS and incorporation flag

### PR DESCRIPTION
**Pull request by richtier**

### CODE REVIEW

File: spire/documents.py [hunk 1]
```diff
@@ -78,6 +78,43 @@ class CLCEntry(InnerDoc):
     )
 
 
+class ClCNestedField(fields.NestedField):
```
Comment (saruniitr) [inline spire/documents.py:81]: `CLCNestedField` ?

File: spire/documents.py [hunk 2]
```diff
@@ -78,6 +78,43 @@ class CLCEntry(InnerDoc):
...
+    def get_value_from_instance(self, instance, field_value_to_ignore=None):
+        value = super().get_value_from_instance(
+            instance=instance, field_value_to_ignore=field_value_to_ignore
+        )
+        if value == "NA":
+            return None
+        return value == "true"
+
+
+class ARSField(fields.KeywordField):
```
Comment (saruniitr) [inline spire/documents.py:100]: as agreed please rename it to `ReportSummaryField`

File: spire/documents.py [hunk 3]
```diff
@@ -78,6 +78,43 @@ class CLCEntry(InnerDoc):
...
+
+
+class IncorporatedBooleanField(fields.BooleanField):
+    def get_value_from_instance(self, instance, field_value_to_ignore=None):
+        value = super().get_value_from_instance(
+            instance=instance, field_value_to_ignore=field_value_to_ignore
+        )
+        if value == "NA":
+            return None
+        return value == "true"
```
Comment (saruniitr) [inline spire/documents.py:97]: is it possible to receive "True", "False"?

**Review by saruniitr — APPROVED** [review]
looks good to me, variable renaming requested

File: spire/documents.py (hunk 3 above)
Comment (richtier) [reply spire/documents.py:97]: nope - I checked against the db - only "true", "false" and "NA" :)

**Pull request merged**